dev-haskell/hackage-security/hackage-security-0.6.0.1.ebuild

```diff
@@ -15,11 +15,9 @@ SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0/${PV}"
 # keep in sync with ghc-8.10
```

Comment:
If keywords are restored the comment is probably out of date.

Reply:
I thought this comment had more to do with which architectures are included in `KEYWORDS`, at least that's how I've interpreted it the handful of times I've seen it.

I mostly left it in to spur a conversation, though, so I'm glad you commented - what should I put as a starting point for `KEYWORDS`? Especially when (as I believe was the case here) hackport starts with a very long list?

Reply:
If you are not sure `~x86 ~amd64` is a good start (assuming they are satisfied).

Reply:
Would it be sufficient to copy the keywords from the previous build, and
then take any out that don't pass repoman and pkgcheck?

I had one (i think cabal) where repoman complained that it couldn't solve
the dependencies for ppc anc ppc64, so I took those out.

Or is it better to "start from scratch" and readd architectures as they're
tested?

I only have an amd64 system to test on, don't have any VM's set up at the
moment.

Reply:
Copying and removing unsatisfied keywords is fine. Starting from scratch usually happens when there is a major change (many new non-trivial depends or build system).

Reply:
Perfect, I'll use this rule-of-thumb moving forward.